pkg/controllers/networkinfo/networkinfo_controller.go

```diff
@@ -252,7 +253,13 @@ func (r *NetworkInfoReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 				deleteFail(r, ctx, obj, &err, r.Client)
 				return common.ResultRequeue, err
 			}
+			ncName, err := r.Service.GetNetworkconfigNameFromNS(obj.Namespace)
+			if err != nil {
+				log.Error(err, "failed to get network config name for VPC when deleting NetworkInfo CR", "NetworkInfo", obj.Name)
+				return common.ResultRequeueAfter10sec, err
+			}
 			log.V(1).Info("removed finalizer", "NetworkInfo", req.NamespacedName)
+			deleteVPCNetworkConfigurationStatus(ctx, r.Client, ncName, vpcs, r.Service.ListVPC())
```

Comment:
seemly, deleteVPCNetworkConfigurationStatus has no error return, if some errors happen in deleteVPCNetworkConfigurationStatus, we will still go to deleteSuccess, is it correct?
do we need to check return error deleteVPCNetworkConfigurationStatus before calling deleteSuccess?

Reply:
There is no error check in `setVPCNetworkConfigurationStatusWithLBS` before `updateSuccess`, I think we need to check both or check neither.

Reply:
ok, if we don't need to check either, that will be ok. but we might need to consider update fail case and retry to make sure keep vpcnetworkconfiguration status updated.